01-js-intro-and-jquery/index.css

```diff
@@ -0,0 +1,13 @@
+.head {text-align: center;}
```

Comment:
center align should be applied to section blocks only but no worries, this is fine.